docs/configuration.md

```diff
@@ -2335,6 +2335,28 @@ spaces.inImportCurlyBraces=true
 import a.b.{c, d}
 ```
 
+### `spaces.inInterpolatedStringCurlyBraces`
+
+```scala mdoc:defaults
+spaces.inInterpolatedStringCurlyBraces
+```
+
+```scala mdoc:scalafmt
+spaces.inInterpolatedStringCurlyBraces = Always
+---
+s"The great ${white} hope"
+s"The racism ${doesn.t} exist"
+s"Four ${more()} years!"
+```
+
+```scala mdoc:scalafmt
+spaces.inInterpolatedStringCurlyBraces = IfComplex
```

Comment:
can you perhaps add some text to explain what `IfComplex` covers?

Reply:
Is it necessary to add IfComplex and Always? We should avoid adding settings based on speculations or what people want.

Reply:
What is your proposal? @olafurpg

Reply:
Can you motivate the need for adding two new settings? Which setting do you personally intend to use? There's a significant maintenance cost to adding a new configurable setting beyond the initial PR, we should not add new settings lightly.

Reply:
>Can you motivate the need for adding two new settings?

To provide more choice of style customization.

>Which setting do you personally intend to use?

`IfComplex`